tests/testfiles/config/sources.php

```diff
@@ -1,16 +1,64 @@
 <?php
 
-$priorities = [
+$mozilla = ['en-US', 'fr', 'it'];
+$slogans_locales = ['fr', 'it'];
+
+$repo0_langfiles = [
     'file1.lang' => [
-        'critical' => ['all'],
-        'obsolete' => ['fr'],
+        'supported_locales' => ['fr'],
+        'deadline'          => '2017-01-30',
+        'flags'             => [
+            'obsolete' => ['fr'],
+        ],
+        'priority' => [
+            1 => ['fr'],
+        ],
     ],
     'file2.lang' => [
-        'critical'  => ['fr'],
-        'obsolete'  => ['all'],
-        'testflag1' => ['all'],
-        'testflag2' => ['fr'],
+        'flags' => [
+            'obsolete'  => ['all'],
+            'testflag1' => ['all'],
+            'testflag2' => ['fr'],
+        ],
+        'priority' => [
```

Comment:
I wonder if it could break if priority for `fr` is set first, and then overridden in the loop by `all`. Maybe it’s worth testing this?
`        'priority' => [`
`            3 => ['fr'],`
`            2 => ['all'],`
`        ],`
And this?
`        'priority' => [`
`            3 => ['it'],`
`            2 => ['all'],`
`            1 => ['fr'],`
`        ],`